CMake/FindSodium.cmake

```diff
@@ -221,8 +221,8 @@ endif()
 
 # extract sodium version
 if(sodium_INCLUDE_DIR)
-  set(_VERSION_HEADER "${_INCLUDE_DIR}/sodium/version.h")
-  if(EXISTS _VERSION_HEADER)
+  set(_VERSION_HEADER "${sodium_INCLUDE_DIR}/sodium/version.h")
```

Comment:
I am curious on motivation for this change, this just sets the sodium version - currently neither in Velox or presto do we enforce a version for sodium. In fact in velox we do not even seem to do find_package or library for sodium. Thus if the sodium version cannot be extracted is it a fatal error ? Also does this findSodium gets invoked via presto-native ?

Reply:
Found this issue while debugging why sodium is not getting linked correctly. This change does not solve the underlying issue.
I can change the version not found to not be a fatal error if that's the convention. This gets called transitively via `find_package(fizz CONFIG)` (which in-turn calls `find_dependency(Sodium)`)